.gcloudignore

```diff
@@ -29,6 +29,7 @@ packager/
 /frontend/
 pages/
 !platform/pages
+/artifacts
```

Comment:
can you rename this to `build` so that we have a single place for all build artifacts.